client/src/pages/CommunityPage/CommunityPagePeople.js

```diff
@@ -228,6 +294,7 @@ const CommunityPage = props => {
             <PersonCard image={image5} name="Julia C." location="Washington D.C." />
           </div>{" "}
           <div className="CommunityPage__momCard-single"> </div> */}
+          {/* {getProfileFromCities} */}
```

Comment:
Go through and delete all commented out code please

Reply:
Removed CommunityPagePeople from commit.